setup.py

```diff
@@ -58,7 +58,7 @@
     ),
     include_package_data=True,  # MANIFEST.in
     install_requires=[
-        "slack_sdk>=3.20.2,<4",
+        "slack_sdk>=3.21.1,<4",
```

Comment:
Set the version to use https://github.com/slackapi/python-slack-sdk/pull/1352 for sure